modules/vector-index/partials/create-composite-index-capella-ui.adoc

```diff
@@ -0,0 +1,91 @@
+====
+. In the *Bucket*, *Scope*, and *Collection* lists, select the bucket, scope, and collection where you have documents that contain a vector field.
+. Under *Vector Field*, in the *Select Field* list, select the vector field to use in your Composite Vector index.
+. (Optional) To add a field to use as a filter on the data included in your Composite Vector index, click btn:[+ Add Filter Field].
+.. Under *Filter Fields*, select the field you want to use to filter your documents before running vector comparisons.
+.. Under *Include in Index*, choose whether the previously selected field should be indexed along with your vector field.
+.. Under *Partition by Hash*, choose whether the previously selected field should be used to partition your index, based on the values in your documents.
++
+For example, you could partition your Composite Vector index based on the value of a `supplier` field in your documents.
+If you choose *No*, you can still filter using a `WHERE` clause, but your index will not be partitioned based on field values.
+.. (Optional) Under *Add WHERE clause*, enter a clause for the specific values that a document must contain in your filter field to be included in your index.
++
+For example, you could filter your index so that only documents with `supplier = "HP"` or `brightness > 0.5` are included.
+
+. Under *Index Configuration*, choose options to configure additional settings for your Composite Vector index: 
+
+.. In the *Similarity Metric* list, select the similarity metric that you want to use for comparing vectors in your index.
+For the best accuracy, use the same similarity that you plan to use to query your data.
++
+For more information about vector similarity measures, see xref:vector-index:vectors-and-indexes-overview.adoc#vector_similarity[Vector Similarity].
+.. In the *Quantization* list, select the type of quantization to use for simplifying the vectors stored in your index.
++
+For more information about quantization, see xref:vector-index:vectors-and-indexes-overview.adoc#quantization[Quantization].
+For more information about how to choose quantization on your index, see xref:vector-index:vectors-and-indexes-overview.adoc#choosing_quantization[Choosing Quantization] and xref:vector-index:vector-index-best-practices.adoc#quantization[Tuning Index Creation - Quantization].
+.. In the *Dimensions* field, enter the exact dimension of the vectors in your data.
++
+Your embedding model determines this value, and Capella automatically configures it based on your chosen *Vector Field*.
+.. In the *Replicas* list, select the number of replicas you want to create for your index.
++
+Replicas affect the query throughput, memory footprint, and fault tolerance for your Composite Vector index.
+More replicas increase the required memory for your index, but can increase query throughput.
+.. In the *Training List* field, enter or select the number of vectors to consider when searching for centroids in your data.
```

Comment:
My understanding is, setting the training list is optional. If you don't specify this value, the Index Service defaults to the values described in the tip below. But you can enter a custom training list value if required.